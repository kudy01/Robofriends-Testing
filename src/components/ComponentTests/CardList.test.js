import { shallow } from 'enzyme';
import React from 'react';
import CardList from '../CardList';

it('Expect to render CardList Component', () => {
	const mockRobots = [
  {
    id: 1,
    name: 'Sameer Khan',
    username: 'Package',
    email: 'ilovetrump.org'
  }]
	expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot()
})