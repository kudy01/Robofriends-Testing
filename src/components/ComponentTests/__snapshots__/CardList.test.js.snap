// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render CardList Component 1`] = `ShallowWrapper {}`;
